app/src/main/java/com/github/sdpteam15/polyevents/model/UserEntity.kt

```diff
@@ -20,70 +22,57 @@ import java.time.LocalDate
  */
 @IgnoreExtraProperties
 data class UserEntity(
-    val uid: String,
+    var uid: String,
     var username: String? = null,
     var name: String? = null,
     var birthDate: LocalDate? = null,
     var email: String? = null,
     var telephone: String? = null,
-    val profiles: MutableList<UserProfile> = mutableListOf()
+    val profiles: MutableList<String> = mutableListOf()
```

Comment:
Because in the db it is a list of profile id which is stored and not the list of profiles itself.
But yes, I will make sure that there is only one request to the db for the profile list